templates/corporate/billing/remote_realm_login_error_for_server_on_active_plan.html

```diff
@@ -21,7 +21,7 @@ <h1 class="lead">{{ _("Plan management not available") }}</h1>
                         organizations on this server. Follow the <a
                         href="https://zulip.com/help/self-hosted-billing#manage-billing">log
                         in instructions</a>
-                        for <b>All older versions</b>
+                        for <b>All versions</b>
                         of the Zulip server to manage your plan.
```

Comment:
I think we might want to do further tweaks to this sentence; I'll do a pass on it shortly and plan to merge it with those tweaks.